app/src/main/res/layout/sound_button.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This will be used for data binding, but it works also without -->
+<!--
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+-->
+    <!-- This will be used for data binding -->
+    <!--<data>
+        <variable
+            name="model"
+            type="data.Sound"/>
+    </data>-->
+<LinearLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:orientation="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+    <!-- TODO: change the default string to something else -->
+    <Button
+            android:id="@+id/sound_button_id"
+            android:text="@string/aiou"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginHorizontal="@dimen/margin_btn_margin_hor_normal"
+            android:layout_marginVertical="@dimen/margin_btn_margin_ver_normal" />
```

Comment:
The margin should change with the size of the layout, I think...

Reply:
This one will be more complicated to fix than I expected...

Reply:
Should I wait for you to fix this before merging or you want to clean this before the merge?

Reply:
I'll wait until @FrancoisDuchene has tested the layouts to inform if it is necessary.